src/V1/Actor/Template/Builder.php

```diff
@@ -32,20 +32,38 @@ public function build(): TemplateInterface
     protected function getFilePath(): string
     {
         if ($this->FilePath === null) {
+            $actorTemplateFilePathCandidates = $this->buildActorTemplateFilePathCandidates();
+            foreach($actorTemplateFilePathCandidates as $actorTemplateFilePathCandidate) {
+                $actorTemplateFilePath = realpath($actorTemplateFilePathCandidate);
+                if ($actorTemplateFilePath !== false) {
+                    $this->FilePath = $actorTemplateFilePath;
+                    return $this->FilePath;
+                }
+            }
+
+            throw new RuntimeException(
+                sprintf('No actor template file was found, checked: %s', implode(',', $actorTemplateFilePathCandidates))
+            );
+        }
+
+        return $this->FilePath;
+    }
+
+    private function buildActorTemplateFilePathCandidates(): array
```

Comment:
method: "I ask a Repository of Template Tree Maps for all Template Trees in storage as a Map. I add the relative file paths. I return an array that is allowed to be empty. I assume that Actors that invoke me ensure the array is valid for their useage"